src/OgAccess.php

```diff
@@ -262,36 +262,37 @@ public function userAccessEntity($operation, EntityInterface $entity, AccountInt
     }
 
     $is_group_content = $this->groupAudienceHelper->hasGroupAudienceField($entity_type_id, $bundle);
-    $cache_tags = $entity_type->getListCacheTags();
-
-    // The entity might be a user or a non-user entity.
-    $groups = $entity->getEntityTypeId() == 'user' ? $this->membershipManager->getUserGroups($entity) : $this->membershipManager->getGroups($entity);
-
-    if ($is_group_content && $groups) {
```

Comment:
We already know at this point whether the entity is group content, so a small reshuffle of these two lines into the if statement is sufficient to not do an unneeded call to `MembershipManager::getGroups()`.